app/serializers/perilune/tasks/index/serializer.rb

```diff
@@ -9,6 +9,7 @@ class Serializer < LedgerSync::Domains::Serializer
         attribute :state
         attribute :tags
         attribute :created_at
+        attribute :file_name
```

Comment:
Needed for identity/documents page since it displays index + show operations on the same page